docs/framework/guides/deep-dive/observables.md

```diff
@@ -0,0 +1,303 @@
+---
+category: framework-deep-dive
+---
+
+# Observables
+
+{@link module:utils/observablemixin~Observable Observables} are objects which have properties that can be observed. That means when the value of such property changes, an event is fired by the observable and the change can be reflected in other pieces of the code that listen to that event.
```

Comment:
It'd be good to explain in this intro where we use observables. In other words – why someone should read this guide and where he/she will be able to apply this knowledge.